build/web_compose/Dockerfile

```diff
@@ -125,7 +125,8 @@ RUN gsutil cp -R gs://datcom-nl-models/ft_final_v20230717230459.all-MiniLM-L6-v2
 WORKDIR /workspace
 
 # Env that should not change
-ENV USE_LOCAL_MIXER=true
+# Local mixer API root
+ENV WEBSITE_MIXER_API_ROOT=http://127.0.0.1:8081
```

Comment:
Can you build and run this both locally and in the cloud? We've run into issues with host changes in the past so want to ensure no regression.

Reply:
Verified locally & in datcom-website-dev